### PR TITLE
fixed some clippy warnings

### PR DESCRIPTION
**Pull request by atk**
Also removed some spaces at eol.

### CODE REVIEW

**Comment by bluss** [comment]
Thanks for working on petgraph. I'm not super stoked for minor clippy fixes (I used it recently and acted on those suggestions I thought were worthwhile).

**Comment by atk** [comment]
If you don't like the changes, feel free to close.

File: src/graphmap.rs [hunk 1]
```diff
@@ -130,11 +130,7 @@ impl<N, E, Ty> GraphMap<N, E, Ty>
     /// Use their natual order to map the node pair (a, b) to a canonical edge id.
     #[inline]
     fn edge_key(a: N, b: N) -> (N, N) {
-        if Ty::is_directed() {
-            (a, b)
-        } else {
-            if a <= b { (a, b) } else { (b, a) }
-        }
+        if Ty::is_directed() || a <= b { (a, b) } else { (b, a) }
```
Comment (bluss) [inline src/graphmap.rs]: Please do, I'll merge it then.

**Comment by bluss** [comment]
Thanks!

**Pull request merged**